packages/manager/apps/hub-redirection/webpack.config.js

```diff
@@ -0,0 +1,73 @@
+const fs = require('fs');
+const glob = require('glob');
+const path = require('path');
+const webpack = require('webpack'); // eslint-disable-line
+const merge = require('webpack-merge');
+const webpackConfig = require('@ovh-ux/manager-webpack-config');
+
+function readNgAppInjections(file) {
```

Comment:
you can remove many things from the config (to be honest i'm not even sure that using webpack is usefull in this case) :
- extras / app injections
- lessPath
- definePlugin
- resolve.modules

Reply:
In fact, dists generated with and without the webpack.config.js are different, so I prefer to change the webpack config file, to have the same with what as been done for former cloud application.